docs/documentation/server_admin/topics/organizations/managing-members.adoc

```diff
@@ -0,0 +1,120 @@
+existing ones.
+
+[[_managed_unmanaged_members_]]
+== Managed and unmanaged members
+
+Before going further on how to manage members, it is important to understand their relationship with an organization given
+that it affects the lifecycle of their accounts. As previously mentioned, members can join an organization through different flows where each of these flows indicates how
+strong is the link between a user and an organization.
+
+There are two types of members:
+
+* *Managed*
+* *Unmanaged*
+
+Managed members are those managed by the organization itself, and they cannot exist without the organization they belong to. For instance,
+an account created through an identity provider associated with an organization does not belong to a realm as it was federated from the organization itself.
+In this case, we can assume that the single-source of truth for the identity is the organization and its lifecycle is controlled
+by the organization.
+Removing the organization or removing the member, should also remove the account from the realm.
+
+On the other hand, unmanaged members are those that can exist without the organization. For instance, when adding an existing
+realm user to an organization, the account belongs to the realm first and foremost and eventually linked to an organization. In this case,
+removing an organization or a member will not remove the account from the realm. In this case, the realm is
+the single-source of truth for the identity.
+
+== Adding an existing realm user as a member
+
+An existing realm user can join an organization by selecting them from a list and adding to the organization.
+
+.Procedure
+
+. Click *Add member*.
+. Click *Add realm user*.
+. Select one or more users and click `Add` to add them to the organization.
+
+Once a member of the organization, a realm user is able to authenticate to the realm just like a regular user and using
+any credential supported by the realm.
+
+== Inviting users
+
+An administrator can email users to join an organization.
+
+.Procedure
+
+. Click *Add member*.
+. Click *Invite member*.
+. Provide an email address
+. Click `Send`
+
+Optionally, you can also provide a value for the `First name` and `Last name` fields for a more personalized email
+message using a greeting message with the first and last names of the person receiving the email.
+
+An invitation is basically an email sent with a link that the person should click to perform the necessary steps to join
+an organization. These steps depend on whether the person already has an account in the realm or if a new account should
+be created before joining the organization.
+
+If the email maps to an existing user in a realm, the steps the user will follow are basically about confirming the
+intention to join the organization.
+
+On the other hand, if there is no user associated with the given email address, the steps
```

Comment:
```suggestion
On the other hand, if no user is associated with the given email address, the steps
```

Reply:
I always try to avoid "there is" or "it is".  Grammatically it doesn't make sense if you ask youself what is the subject of the sentence.  Is it "there?"

Reply:
Fixed by https://github.com/keycloak/keycloak/pull/29796/commits/ba047c2ee5710833ab08c7df71b2116d5b1cda41.